### PR TITLE
feat: add initial equalized deploy command

### PR DESCRIPTION
**Pull request by PatrickMenoti**
Once merged, this PR will generate a new major (2.0).

### CODE REVIEW

File: .github/workflows/test_vulcan.yml [hunk 1]
```diff
@@ -30,6 +30,18 @@ jobs:
       - name: Setting up a secure repository
         run: git config --global --add safe.directory /__w/azion/azion
 
-      - name: test vulcan
+      # - name: test vulcan
```
Comment (maxwelbm) [inline .github/workflows/test_vulcan.yml:33]: it would be interesting to remove this comment

File: messages/edge_storage/messages.go [hunk 2]
```diff
@@ -66,9 +66,9 @@ const (
 	ASK_NOT_EMPTY_BUCKET           = "Your bucket can't be deleted because it's not empty. Do you want to empty it? (y/N)"
 
 	OUTPUT_CREATE_BUCKET = "Bucket created successfully"
-	OUTPUT_DELETE_BUCKET = "Bucket %s was deleted successfully"
-	OUTPUT_UPDATE_BUCKET = "Bucket updated successfully"
-	OUTPUT_UPDATE_OBJECT = "Object updated successfully"
-	OUTPUT_CREATE_OBJECT = "Object created successfully"
-	OUTPUT_DELETE_OBJECT = "Object %s was deleted successfully"
+	OUTPUT_DELETE_BUCKET = "Bucket %s was deleted successfully\n"
```
Comment (maxwelbm) [inline messages/edge_storage/messages.go:69]: Wouldn't it be more interesting to use `\n` in the output print to make it work for everyone?

File: pkg/cmd/deploy/upload.go [hunk 3]
```diff
@@ -15,21 +17,26 @@ import (
...
-	if err := cmd.FilepathWalk(PathStatic, func(path string, info os.FileInfo, err error) error {
+	if err := cmd.FilepathWalk(pathStatic, func(path string, info os.FileInfo, err error) error {
 		if err != nil {
 			logger.Debug("Error while reading files to be uploaded", zap.Error(err))
-			logger.Debug("File that caused the error: " + PathStatic)
+			logger.Debug("File that caused the error: " + pathStatic)
 			return err
 		}
+
+		// Skip node_modules and .edge directories
```
Comment (maxwelbm) [inline pkg/cmd/deploy/upload.go:34]: NICE

**Review by maxwelbm — APPROVED** [review]

**Comment by maxwelbm** [comment]
Lgtm :)

File: .github/workflows/test_vulcan.yml (hunk 1 above)
Comment (rodrigomaria) [inline .github/workflows/test_vulcan.yml:33]: Does needed this comment?

Comment (PatrickMenoti) [reply .github/workflows/test_vulcan.yml:33]: I can remove it... both you and Max pointed it out!

**Review by rodrigomaria — APPROVED** [review]

**Pull request merged**